tests/unit/helpers/is-feature-enabled-test.js

```
@@ -0,0 +1,37 @@
+import config from 'frontend-loket/config/environment';
+import isFeatureEnabled from 'frontend-loket/helpers/is-feature-enabled';
+import { module, test } from 'qunit';
+
+module('Unit | Helper | is-feature-enabled', function () {
+  test('it works', function (assert) {
+    let originalFeaturesConfig = config.features;
+
+    config.features = {
+      foo: 'bar',
+    };
+
+    assert.false(
+      isFeatureEnabled('foo'),
+      "it returns false if the value isn't the boolean `true` or the string 'true'"
+    );
+
+    config.features.foo = true;
+    assert.true(
+      isFeatureEnabled('foo'),
+      'it returns true if the value is the boolean `true`'
+    );
+
+    config.features.foo = 'true';
+    assert.true(
+      isFeatureEnabled('foo'),
+      "it returns true if the value is the string 'true'"
+    );
+
+    assert.false(
+      isFeatureEnabled('baz'),
+      "it returns false if the feature name doesn't exist"
```

Should we throw an error during development instead?

---

I went with the assert version since that will help us to find typos more easily.